src/platform/forms-system/src/js/utilities/benefit-description.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @typedef {Object} BenefitDescription
+ * @prop {string} benefitType - The type of benefit; e.g. 'health care benefits'
+ * @prop {string|boolean} [subText=] - The subtext to be wrapped in (). e.g.
+ *                             '10-10EZ'. Defaults to the formId. If set to
+ *                             false or an empty string, the subtext will be
+ *                             omitted from the benefit description text.
+ */
+
+/**
+ * Return the benefit description string in the following format:
+ * 'benefit description (subtext or form ID)'
+ *
+ * @param {BenefitDescription} formConfig.benefitDescription
+ * @param {string} formConfig.formId
+ * @returns {string} - The benefit description as a string;
+ *                     e.g. 'health care benefits (10-10EZ)'
+ */
+export default function getBenefitString(formConfig) {
```

Comment:
This and `getSavedFormDescription` feel super clunky. The idea was to mimic the current functionality, but I'm not convinced that's the best approach. Not sure what would be better, though. I'm wide open to suggestions.

The main thing is that the text needs to come from the `formConfig` in some way so we're not importing non-forms library code into the forms library (such as the personalization dashboard helpers).